src/pages/create-account/CreateAccountPage.tsx

```diff
@@ -0,0 +1,133 @@
+    onFieldValueChange 
+  } = useForm(formDefinition);
+  
+  const buildRequestBody = () => ({
+    email: formState.get('email').value as string,
+    password: formState.get('password').value as string,
+  });
+
+  const submitCreateAccountForm = () => {
+    if (!isFormValid) return;
+
+    setState(OperationState.LOADING);
+    setErrorMessage('');
+
+    createAccount(buildRequestBody())
+      .then(() => {
+        setState(OperationState.SUCCESS);
+      })
+      .catch((error) => {
+        setState(OperationState.INPUT);
+        // TODO: This error message is temporary. For debugging.
+        setErrorMessage('message: ' + error.message + ', code: ' + error.code);
+        console.log(error);
+      });
+  };
+
+  const navigateToLoginPage = () => {
+    navigate('/login');
+  };
+
+  return (
+    <div className={styles.createAccountFormContainer}>
+      <AgarLogo 
+        className={styles.logoContainer}
+        ariaLabel='agartex logo'
+        testId='create-account-agartex-logo'/>
+
+      { state === OperationState.INPUT && (
+        <>
+          <div className={styles.createAccountFormHeader}>
+            Create new account
+          </div>
+
+          <TextInput
+            ariaLabel='email text field'
+            testId='create-account-email-text-input'
+            initialValue={formState.get('email').value as string}
+            placeholder='Enter your email'
+            type='text'
+            onChange={(val) => onFieldValueChange('email', val)}
+            onFocus={() => onFieldTouch('email')}
+            isValid={!isInErrorState('email')}
+            errorMessage='Email is invalid'
+          />
+
+          <TextInput
+            ariaLabel='password text field'
+            testId='create-account-password-text-input'
+            initialValue={formState.get('password').value as string}
+            placeholder='Enter your password'
+            type='password'
+            onChange={(val) => onFieldValueChange('password', val)}
+            onFocus={() => onFieldTouch('password')}
+            isValid={!isInErrorState('password')}
+            errorMessage='Password does not meet requirements'
+          />
+
+          <Button
+            ariaLabel='create account submit button'
+            testId='create-account-submit-button'
+            value='Submit'
+            onClick={submitCreateAccountForm}
+          />
+        </>)
+      }
+      { state === OperationState.SUCCESS && <CreateAccountSuccessBox /> }
+      { state === OperationState.LOADING && 
+        <LoadingSpinner 
+          ariaLabel='create account loading spinner'
+          testId='create-account-loading-spinner'
+          className={styles.loadingSpinner}
+        />
+      }
+      { state !== OperationState.LOADING && 
+        <Button 
+          ariaLabel='back to login button'
+          testId='back-to-login-button'
+          value='Go back to login'
+          onClick={navigateToLoginPage}
+        />
+      }
```

Comment:
This is also considered as submit button, so the 'Save password?' prompt appears in my browser. Is it possible to fix it somehow?

Reply:
1. I changed form buttons to disable instead of hiding on loading
2. @hubzie  i changed these buttons' types to 'button', can you check if this still happens to you? I wasn't able to reproduce it

Reply:
I still observe this bug. Let's leave it and maybe one I will find the cause